### DISCUSSION TITLE
What is the original bit depth of images in the dataset?

### DISCUSSION THREAD

**Discussion by crsanderford** [facebookresearch/fastMRI#319] [Q&A] [answered]
Hi all. I'm using the FastMRI dataset for training several models in PyTorch but I'm forced to use a small batch size to fit whole scans into memory. I'm considering reducing precision on the data tensors, and my understanding is that the DICOM format never exceeds a bit depth of 16 per pixel, which makes reducing precision appealing. Is this true of the FastMRI dataset as well?

**Comment by mmuckley** [comment, marked as answer]
Hello @crsanderford, I think the data that comes off the scanner is usually of short precision. Each vendor will have some components of how they do the quantization to maximize the signal value over the short range. Since the shorts don't usually mean anything, they are divided by the maximum value and the raw data is saved as floats, which we prefer to work with as the physics math is continuous. Then, there is another quantization step when saving to the DICOM.

So the answer is: "maybe". Probably most papers take the floating point version of the raw data, work with that, and report all their metrics using floating point data (ignoring the quantization issues of both raw data collection and DICOM file writing), so from an academic point of view these aspects of quantization are not very well studied and you may be diverging from the field.

**Reply by crsanderford** [reply]
Understood, thank you!
